kevm-pyk/src/tests/conftest.py

```diff
@@ -28,6 +28,6 @@ def update_expected_output(request: FixtureRequest) -> bool:
     return request.config.getoption('--update-expected-output')
 
 
-@pytest.fixture(scope='module')
+@pytest.fixture(scope='session')
```

Comment:
I see another fixture with the `module` scope [here](https://github.com/runtimeverification/evm-semantics/blob/master/kevm-pyk/src/tests/integration/test_prove.py#L165). Is this one ommited on purpose?

Reply:
It might be able to be changed, but it's outside the scope of these changes. The reason this one was changed was because it was a dependency of the other one.